conf/machine/hikey-32.conf

```diff
@@ -0,0 +1,45 @@
+#@TYPE: Machine
+#@NAME: LeMaker HiKey machine
+#@DESCRIPTION: LeMaker HiKey machine
+
+require conf/machine/include/tune-cortexa8.inc
+DEFAULTTUNE = "cortexa8hf-neon"
```

Comment:
Setting the preferred floating point format is a DISTRO thing, not a MACHINE thing. Your comment illustrates perfectly why this DEFAULTTUNE needs to get removed from the machine.conf.

Wrt the mali binaries, I guess meta-rpb needs to import this: https://github.com/Angstrom-distribution/meta-angstrom/blob/master/conf/distro/include/arm-defaults.inc